portfolio/src/main/webapp/index.html

```diff
@@ -17,15 +22,18 @@ <h2>Hello everyone, I'm Leonard Zhang. You can call me Leonard. I’m a rising j
       <h2> Also, I really enjoy hiking and travelling with my family members during the vacations. My favorite destination
           is Japan and I have visited there twice in the past three years.</h2>
       <img id="first-picture" src="/images/Leonard-3.jpg"/>
+      <p class="img-description">I took this picture at Shiraito Falls (白糸の滝), Fujinomiya, Japan.</p>
       <h2>Because of my interest in games, I love developing some small games with my friends during my spare time. It’s 
           such a great fun playing the games developed by my own. This spring, I developed a board game called carcassonne
           with my friends.</h2>
       <img id="carcassonne" src="/images/carcassonne.png"/>
-      <p>Click to learn a random fact about me!</p>
+      <p class="img-description">This is the game Carcassonne that my friends and I developed using Java this spring.</p>
```

Comment:
HTML has structures for image captions that you can use: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption

Reply:
Thank you so much! This is much easier than what I've written. I will adapt this in the new commit.

Reply:
Sounds good - looks like you're still pushing commits, please let me know when it's ready for review again.

Reply:
I've just pushed the latest commit. I think it's ready for review now. : )